package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "license-checker-github-action",
```

Comment:
Perhaps we can get rid of the `github` part in the name and repo as it's quite redundant or obvious.